docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,329 @@ Review important information about the {kib} 8.x releases.
+The Kafka output no longer supports conditional topics. For more information, refer to ({kibana-pull}176879[#176879]).
+====
+      
+[discrete]
+[[breaking-176443]]
+.Most Fleet installed integrations are now read-only and labelled with a *Managed* tag in the Kibana UI 
+[%collapsible]
+====
+*Details* +
+
+Integration content installed by {fleet} is no longer editable. This content is tagged with *Managed* in the {kib} UI, and is Elastic managed. This content cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned. The clones can be customized.
+
+When cloning a dashboard the cloned panels become entirely independent copies that are unlinked from the original configurations and dependencies. 
+
+Managed content relating to specific visualization editors such as Lens, TSVB, and Maps, the clones retain the original reference configurations. The same applies to editing any saved searches in a managed visualization.
+
+For more information, refer to ({kibana-pull}172393[#172393]).
+====
+  
+[discrete]
+[[breaking-174674]]
+.Removes `is_nan`, `is_finite`, and `is_infinite` functions from {esql}
+[%collapsible]
+====
+*Details* +
+These functions have been removed from {esql} queries as they are not supported. Errors would be thrown when trying to use them. For more information, refer to ({kibana-pull}174674[#174674]).
+====
+      
+[float]
+[[features-8.13.0]]
+=== Features
+{kib} 8.13.0 adds the following new and notable features.
+
+Alerting::
+* The Custom Threshold rule is now out of technical preview and generally available ({kibana-pull}176514[#176514]).
+* Adds threshold to the custom threshold alert document ({kibana-pull}176043[#176043]).
+* Adds the ability to post Block Kit messages to the Slack Web API action ({kibana-pull}174303[#174303]).
+* Adds criticality fields and risk score fields to alert schema ({kibana-pull}174626[#174626]).
+* Adds fields table to rule details page alert flyout ({kibana-pull}172830[#172830]).
+APM::
+* Show Universal Profiling on Transaction view ({kibana-pull}176302[#176302]).
+* Adds a specific metrics dashboard for opentelemetry-node services ({kibana-pull}174700[#174700]).
+Elastic Security::
+For the Elastic Security 8.13.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Adds reference to `ecs@mappings` for each index template ({kibana-pull}174855[#174855]).
+* Adds support for the `subobjects` setting on the object type mapping ({kibana-pull}171826[#171826]).
+Infrastructure::
+* Adds a new Services component to host details UI ({kibana-pull}176539[#176539]).
+Integrations::
+* Adds tiles for Notion and Redis connectors ({kibana-pull}177306[#177306]).
+Lens & Visualizations::
+* Adds the creating of {esql} charts from the dashboard ({kibana-pull}171973[#171973]).
+* Adds enhanced {esql} query editing experience with client side validation ({kibana-pull}170071[#170071]).
```

Comment:
Let's create a new ES|QL section and add this because this is a feature which corresponds everywhere the ES|QL editor exists and not only on visualizations

Reply:
That makes a lot of sense, thank you!